RussianFilter/sections/specific.txt

```diff
@@ -91,6 +91,8 @@ mirtesen.ru##.adfox-outer
 orsk.ru##.banner-top
 orsk.ru##div[bs_id]
 orsk.ru##.bs_bottom
+music-torrent.com##div[style="text-align: center"] > a[href^="engine/go.php"]
+||music-torrent.com/templates/fast/images/ban_*.gif
```

Comment:
It's better to combine these ad gifs to one rule via Regex.
Also you can use the cosmetic rule above, but in this case you don't need to block them with requests.